blog/2020-10-23-the-making-of-new-user-sdk-tutorials.md

```diff
@@ -0,0 +1,110 @@
+
+Once it became clear that a standard approach wasn't going to cut it, we approached our co-founder and CEO, Maxim Fateev. We asked him to give us a demonstration that he typically uses to introduce engineers to Temporal for the first time. The scenario represents a money transfer from one bank account to another and he demonstrates what happens if one of the steps in the transfer fails. The money transfer sample was a great way to introduce the values of Temporal. For if you understand the ramifications of losing money from a failed financial transaction, several values of Temporal become immediately apparent:
+
+1. The state of running code is maintained even through hardware failures, server crashes, and network outages.
+2. There is deep visibility into the state of code execution out of the box via the CLI or UI.
+3. Function calls come with automatic and retries and configurable timeouts.
+4. Bugs can be hot fixed in running code.
+
+For someone that is new to Temporal the attraction doesn’t come from using the SDK to print “Hello World!”, but instead comes from witnessing the inherent benefits that Temporal offers by simply running simulations using a pre-built application.
+
+This is the direction that we decided to send new users. If a user can wrap their heads around the value that Temporal brings to their application right out of the box, then spending time and energy on application setup and learning the SDK becomes a non-blocker.
+
+Check out these finished tutorials and see for yourself!
+
+- [Go: Run your first Temporal application](/docs/go-run-your-first-app)
+- [Java: Run your first Temporal application](/docs/java-run-your-first-app)
+
+## Next steps
+
+<img alt="image" class="docs-image-centered docs-image-max-width-20" src="https://raw.githubusercontent.com/temporalio/documentation-images/main/static/steps.png" />
+
+At Temporal we understand that our documentation plays a very important role in our users' experience. And to get our docs into a world class state we have lots of work in front of us. In our next steps we will be looking at the end-to-end journey through our documentation and how we can provide the best experience for every user. To validate any direction we take we will be engaging with the community to hear what you think and help us dial things in. Any user can [schedule a 15 minute feedback session](https://calendly.com/cully-temporal/schedule-feedback-session) with me directly! We will also be preparing for all the new and exciting features around our hosted cloud offering that will enable all developers to build invincible applications.
```

Comment:
"In our next steps" awkward phrasing IMO

"To validate any direction we take we will be engaging with the com" - comma